### DISCUSSION TITLE
Welcome to goneovim Discussions!

### DISCUSSION THREAD

**Discussion by akiyosi** [akiyosi/goneovim#185] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by justinmcgrath** [comment]
I can't figure out how to activate smooth scrolling. I've created `~/.goneovim/settings.toml` and added the following:
```
[Editor]
SmoothScroll = true
```

I start goneovim using this command:
`goneovim.exe --nvim="..\Neovim\bin\nvim.exe" -u NONE`

Using `-u NONE` to disable smooth scrolling plugins I have. `ctrl-D` and `ctrl-u` do not have smooth scrolling though.

This works though.
```
[Cursor]
SmoothMove = true
```

Is there something else I need to do to get smooth scrolling?

**Reply by akiyosi** [reply]
Hi :D
Does  `SmoothMove` one work?
Hmm, that's odd.
Could you please tell me the full contents of your settings.toml?

**Reply by justinmcgrath** [reply]
Thanks. Here are the contents.
```
[Editor]
SmoothScroll = true

[SideBar]
Visible = true

[Cursor]
SmoothMove = true
```

SmoothMove works. I added the SideBar just to see if it works, and it does. Only SmoothScroll doesn't work.

**Reply by akiyosi** [reply]
Umm, what version of nvim.exe are you using?
Goneovim currently recommends Neovim 0.5.0.
If you have an older Nvim, the GUI events needed for smooth scrolling may not be fired.

**Reply by justinmcgrath** [reply]
That was it. Thank you.
